README.rst

```diff
@@ -0,0 +1,152 @@
+Elm Docstyle
+============
+A tool that allows you to check the existence and quality of your `Elm <http://elm-lang.org/>`_ code comments.
+
+Usage
+=====
+
+Prerequisites
+
+
+The following binaries should be available on the path:
+
+
+.. code-block:: bash
+
+    node >=6
+    elm  0.18.x
+
+
+Installation
+============
+
+With ``npm``:
+
+.. code-block:: bash
+
+    npm install -g elm-docstyle
+
+or if using ``yarn``:
+
+
+.. code-block:: bash
+
+    yarn global add elm-docstyle
+
+
+Usage
```

Comment:
I'd move usage _before_ installation since it actually presents what features the CLI provides.

Reply:
I feel like the usage paragraph is a bit too long (especially if we consider the "configuration" which should follow right after it). If that's okay by you, I'd follow [elm-format](https://github.com/avh4/elm-format)'s convention and add a "Usage" paragraph at the beginning with a couple examples of an invocation, followed by the short "Installation" paragraph and finally the "detailed instructions".